app/BitSet.js

```diff
@@ -30,7 +30,11 @@ BitSet = function (nBitsOrKey) {
       for (i = 0; i < arrVals.length; i++) front[leadingZeros + i] = arrVals[i];
       arrVals = front;
     }
-    this.arr = new Uint32Array(arrVals);
+    wordCount = Math.ceil((this.MAX_BIT + 1) / BITS_PER_INT);
+    this.arr = new Uint32Array(wordCount)
+    for (var i = 0; i < arrVals.length; i++) {
```

Comment:
Sure, no need for a squash or anything, you should be able to patch it
straight from github.

On Wed, Mar 9, 2016, 8:06 PM Million <notifications@github.com> wrote:

> In app/BitSet.js
> <https://github.com/mattkrick/fast-bitset/pull/8#discussion_r55624872>:
>
> > @@ -30,7 +30,11 @@ BitSet = function (nBitsOrKey) {
> >        for (i = 0; i < arrVals.length; i++) front[leadingZeros + i] = arrVals[i];
> >        arrVals = front;
> >      }
> > -    this.arr = new Uint32Array(arrVals);
> > +    wordCount = Math.ceil((this.MAX_BIT + 1) / BITS_PER_INT);
> > +    this.arr = new Uint32Array(wordCount)
> > +    for (var i = 0; i < arrVals.length; i++) {
>
> Yep, u r right, should I replace the loop and PR again?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mattkrick/fast-bitset/pull/8/files#r55624872>.
>